docs/develop/common-coding-issues.md

```diff
@@ -138,6 +138,10 @@ The following APIs are affected by this workbook switch:
 > [!NOTE]
 > This only applies to multiple Excel workbooks open on Windows or Mac.
 
+### Coauthoring
+
+See [Coauthoring in Excel add-ins](../excel/co-authoring-in-excel-add-ins.md) for patterns to use with events in a coauthoring environment. The article also discusses potential merge conflicts when using certain APIs, such as [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-).
```

Comment:
The second link in line 143 is broken when I test it on the review site. Extra dash at the end?

Reply:
Review links often break when going between repos because the branch doesn't exist in the other repo. You can test the link by removing the "review" part after navigation.